--- Makefile.old
+++ Makefile
@@ -814,7 +814,7 @@
 endif
 endif
 
-install: finish
+install:
 	mkdir -p "$(INSTALL_DIR)"
 	cp -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
 
@@ -920,6 +920,8 @@
 	    -U__wasm_nontrapping_fptoint__ \
 	    $(if $(filter-out expected/wasm32-wasip1-threads,$(EXPECTED_TARGET_DIR)),-U__wasm_bulk_memory__) \
 	    -U__wasm_bulk_memory_opt__ \
+	    -U__wasip1__ \
+	    -U__wasip2__ \
 	    -U__GNUC__ \
 	    -U__GNUC_MINOR__ \
 	    -U__GNUC_PATCHLEVEL__ \
